examples/run_CPMG.py

```diff
@@ -1,83 +1,72 @@
+import configparser
 from pylab import *
 from pyspecdata import *
 from numpy import *
 import SpinCore_pp 
+from config_parser_fn import parser_function
 from datetime import datetime
 fl = figlist_var()
 
+#{{{importing acquisition parameters
+values, config = parser_function('active.ini')
+nPoints = int(values['acq_time_ms']*values['SW_kHz']+0.5)
+#}}}
+#{{{create filename and save to config file
 date = datetime.now().strftime('%y%m%d')
-output_name = 'TEMPOL_capProbe'
-node_name = 'CPMG_4step_6'
-adcOffset = 25
-carrierFreq_MHz = 14.895548
-tx_phases = r_[0.0,90.0,180.0,270.0]
-amplitude = 1.0
-p90_us = 4.477
-deadtime_us = 10.0
-repetition_us = 12e6
-deblank_us = 1.0
+config.set('file_names','type','CPMG')
+config.set('file_names','date',f'{date}')
+values['echo_counter'] += 1
+config.set('file_names','echo_counter',str(values['echo_counter']))
+config.write(open('active.ini','w')) #write edits to config file
+values, config = parser_function('active.ini') #translate changes in config file to our dict
+filename = f"{values['date']}_{values['chemical']}_{values['type']}_{values['echo_counter']}"
```

Comment:
not sure if this will work as it's not tested, but I know the previous were struggling to combine a str and int, where int is the date. Since the other version I use in combined_ODNP.py definitely works I will use that format for the other scripts